template.config.json

```diff
@@ -250,7 +250,8 @@
         }
     },
     "data_sources": {
+        "curated_loras_source": "https://raw.githubusercontent.com/Haidra-Org/AI-Horde-image-model-reference/main/lora.json",
         "styles_source": "https://raw.githubusercontent.com/Haidra-Org/AI-Horde-Styles/main/styles.json",
-        "style_categories_source": "https://raw.githubusercontent.com/db0/Stable-Horde-Styles/main/categories.json"
+        "style_categories_source": "https://raw.githubusercontent.com/Haidra-Org/Stable-Horde-Styles/main/categories.json"
```

Comment:
please make sure that the link is actually valid, in this case it is not valid and returns a 404